src/components/WebexMemberRoster/WebexMemberRoster.jsx

```diff
@@ -23,12 +23,32 @@ import useMemberships from '../hooks/useMemberships';
  * @returns {object} JSX of the component
  */
 export default function WebexMemberRoster({destinationID, destinationType}) {
-  const memberships = useMemberships(destinationID, destinationType);
-  const members = memberships.map(
+  const members = useMemberships(destinationID, destinationType);
+
+  const renderMembers = (members) => (members || []).map(
     (member) => <WebexMember key={member.personID} personID={member.personID} />,
   );
 
-  return <div className={`${WEBEX_COMPONENTS_CLASS_PREFIX}-roster`}>{members}</div>;
+  if (destinationType !== 'meeting') {
+    return <div className={`${WEBEX_COMPONENTS_CLASS_PREFIX}-roster`}>{renderMembers(members)}</div>;
+  }
+
+
+  const inMeetingMembers = members.filter(member => member.isInMeeting);
+  const notInMeetingMembers = members.filter(member => !member.isInMeeting);
+
+  return (
```

Comment:
The problem with this approach, as demonstrated in the description video, is that while there are no members loaded, the user sees the shrunk headers first. We should display all or nothing, maybe a spinner while we gather the data.